k8s/pkg/schedulers/scheduling_strategy.go

```diff
@@ -54,6 +54,9 @@ func NewRoundRobinStrategy(nodes []string) *RoundRobinStrategy {
 // Compute returns the node by the given rank.
 func (r *RoundRobinStrategy) Compute(rank int) (string, error) {
 	l := len(r.nodes)
+	if l == 0 {
+		return "", nil
+	}
```

Comment:
Should we return an error if `nodes` is empty?

Return nil (success) with a invalid result `""` looks quite strange.

Reply:
Yeah, updated. Returning an error message is useful for debugging.